libraries/stdlib/samples/test/samples/number/int.kt

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2010-2017 JetBrains s.r.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package samples.number
+
+import samples.*
+
+class Int {
+
+    @Sample
+    fun coerceAtLeast() {
+        assertPrints(10.coerceAtLeast(5), "10")
+        assertPrints(10.coerceAtLeast(20), "20")
```

Comment:
Do you want to add more sample code for other primitives? @ilya-g

Reply:
I don't think it would significantly improve the sample. On the other hand we could add a separate sample for some comparable types other than numbers.

Reply:
Have you succeeded with this? I mean a sample for some comparable type.